index.md

```diff
@@ -43,19 +43,19 @@
 | bufo-alarma.gif | ![bufo-alarma.gif](all-the-bufo/bufo-alarma.gif) |
 | bufo-all-good.png | ![bufo-all-good.png](all-the-bufo/bufo-all-good.png) |
 | bufo-all-warm-and-fuzzy-inside.png | ![bufo-all-warm-and-fuzzy-inside.png](all-the-bufo/bufo-all-warm-and-fuzzy-inside.png) |
-| bufo-am-i.png | ![bufo-am-i.png](all-the-bufo/bufo-am-i.png) |
```

Comment:
Interesting. My system is ordering dashes after alphas in the generated file. Will revert this for now.